### ISSUE TITLE
[Placeholder / Discussion] Draw a picture of what the QC report will contain for MVP

### ISSUE THREAD

**Issue by jaclyn-taroni** [AlexsLemonade/scpcaTools#37]
A good way to make sure we're all on the same page about the contents and layout of the QC report is to sketch out what the overall report will look like. 

From there, we can 1. disagree a little bit initially, probably 2. ensure we have issues for all the relevant sections 🚀

**Comment by allyhawkins** [comment]
In an effort to get this started I went ahead and created a very very rough drawing of what I think we are looking for in the QC report. The first two tables, `sample_information` and `processing_information` we already have a version added to the report so I just filled in some examples of data that is in there but it is not complete so as not to crowd the picture. 

I also tried my best to draw out the plots but in general, I think we would have the following plots: 
- our usual plot we show during training with UMI/cell on the x-axis, genes/cell on the y-axis and colored by mito content 
- knee plot where we are coloring cells based on if they are included in the filtered sce object or not (filtering here referring to `emptyDrops` filtering) 
- plot showing the miQC model with unique genes on the x-axis and mito on the y-axis and showing the model. Would we rather change this and maybe color the cells based on if they are keep or throw based on miQC? 

I also added in a section for ADT/ CITE-seq information and noticed on the [issue](https://github.com/AlexsLemonade/scpcaTools/issues/38) that there was a comment to add in the list of antibodies. I wasn't sure how lengthy that could get or how we would want to portray that so I thought we might want to have that separate from the statistics? 

I just wanted to get a general idea of the order of things and the blocks that we need to put in place so this is pretty rough but would love to hear @jaclyn-taroni and @jashapiro thoughts/ opinions/ disagreements/ suggestions. I also wanted to keep in mind that this is for MVP so I did not choose to include any UMAPs or clustering at this point. 

<img width="397" alt="Screen Shot 2021-09-24 at 4 32 20 PM" src="https://user-images.githubusercontent.com/54039191/134741919-276d5d8f-c28f-48f3-8493-cd1482807c6c.png">

**Comment by jaclyn-taroni** [comment]
This seems perfectly reasonable to me for MVP, which is to say I have no disagreements. I'm uploading a version of that sketch with lettered panels just in the interest of making the next steps easier to talk about.

<img width="397" alt="134741919-276d5d8f-c28f-48f3-8493-cd1482807c6c" src="https://user-images.githubusercontent.com/19534205/134955140-cf93b960-3491-4261-8b8b-f7c9e82fa57d.png">


If @jashapiro also thinks this seems reasonable, I think the create/update issues part of this is as follows (I'm working from [`inst/rmd/qc_report.rmd`](https://github.com/AlexsLemonade/scpcaTools/blob/fa33a5a81e806682a91e77274bbbb9b81fa96571/inst/rmd/qc_report.rmd)):

- Identify a way to make a "two-column" report. If there is no reasonable solution to this problem, revisit the organization using a single column.
- Ensure **A** ("Raw Sample Metrics") has all the information we need already. If not, file an issue with what needs to be added/updated. 
- Ensure **B** ("Pre-processing Information") has all the information we need already. If not, file an issue with what needs to be added/updated.
- Update #18 to reflect **C**
- Update #39 to reflect **D**
- Ensure **E** ("Knee plot") is done. If not, file an issue with what needs to be added/updated. (Can we close #19?)
- Create an issue for **F**
- Update and/or split up #38 to reflect **G** & **H**

Issues should have enough implementation details to get folks headed in the right direction. If we make them more specific than this sketch, which we should aim for, we can have any disagreements about _details_ there.

**Comment by jashapiro** [comment]
I looked into the two column thing: Looks like we can do that with "fenced divs" https://bookdown.org/yihui/rmarkdown-cookbook/multi-column.html

Basically we add a bunch of `::: {}` elements where we want to create divs, and enclose ones we want side by side with `::::{style="display: flex;"}` 

WE can try it at least!

**Comment by jaclyn-taroni** [comment]
👍🏻 new issue please :)

**Comment by jashapiro** [comment]
I do wonder if we should combine A&C? Or move the number of cells to C? Conceptually and statistically, the number of cells after filtering and UMI/cell are related, so it seems like we might want to keep those together.

**Comment by allyhawkins** [comment]
Right now we have the following included in A: 
- Sample ID
- Tech version 
- Number of reads sequenced
- Number of mapped reads
- Number of cells reported by alevin-fry 

The statistics mentioned in #18 that would be part of the table in C include median UMI/cell, median genes detected/cell, and median mito %/cell. We could append that to what we have currently in A? or We could take `Number of cells reported by alevin-fry` and move that to C and keep A as a separate table with information about the whole sample rather than at a cellular level?

**Comment by jashapiro** [comment]
Ah, okay.. I was misunderstanding the # of cells as the number after filtering... but it is only after the barcode filtering, not emptyDrops. I think that is fine to leave in A. 

We should add # of cells post-emptyDrops to #18, and I will add that there.

**Comment by allyhawkins** [comment]
> We should add # of cells post-emptyDrops to #18, and I will add that there.

Agree with this, I also forgot that we included the pre empty-drops number of cells when we added that table for A to the QC report originally. Sorry about the confusion.

**Comment by jashapiro** [comment]
Closing for future work.
